csharp/src/Drivers/Databricks/DatabricksParameters.cs

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Arrow.Adbc.Drivers.Databricks
+{
+    /// <summary>
+    /// Parameters used for connecting to Databricks data sources.
+    /// </summary>
+    public class DatabricksParameters
```

Comment:
Should this be a static class like `SparkParameters`? Conversely, does it make sense for the `Parameters` to be non-static and then have `DatabricksParameters` derive from `SparkParameters`?

Reply:
That works. I was trying to separate the two, but I get the idea. I made that adjustment in the latest PR.